### DISCUSSION TITLE
Reset command “not approved”

### DISCUSSION THREAD

**Discussion by zimbor** [gopro/labs#962] [Q&A]
Hi. I’ve been using labs firmware for a bit now and it seems to be working fine. However when I try and use the Command !RESET!10 I get a not approved on the GoPro display. I’m not sure what the 3 button press message is either. Please advise and thanks.

**Comment by dnewman-gpsw** [comment]
It is asking for three button presses, you need to press shutter or mode button three times.  This is so someone can't erase your settings without your approval.

**Reply by zimbor** [reply]
Thanks. It’s the shutter button for me. Didn’t see that in the docs. Now a stupid question of mine…😑
